src/cache/chrome-storage-async-map.ts

```diff
@@ -0,0 +1,24 @@
+import { AsyncMap } from '@eppo/js-client-sdk-common';
+
+/** Chrome storage-backed {@link AsyncMap}. */
+export default class ChromeStorageAsyncMap implements AsyncMap<string, string> {
```

Comment:
this is generic enough that it can be shared by both configuration and assignment caches